File: Semester2/OOP/t1/domain/AppException.cpp
//
// Created by georgerapeanu on 07.04.2022.
//

#include "AppException.h"

AppException::AppException() {
    this->message = "";
}

AppException::AppException(std::string message) {
    this->message = message;
}

std::string AppException::get_message() {
    return this->message;
}

File: Semester4/Systems for Design and Implementation/backend/src/main/java/com/example/app/exceptions/AppException.java
package com.example.app.exceptions;

/**
 * An exception specific to the application.
 */
public class AppException extends Exception {
    /**
     * Constructs a new AppException with the specified detail message.
     *
     * @param message the detail message
     */
    public AppException(String message) {
        super(message);
    }
}
